priv/repo/migrations/20240408112012_add_multiple_tenants_to_databases.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Trento.Repo.Migrations.AddMultipleTenantsToDatabases do
+  use Ecto.Migration
+
+  def change do
+    alter table(:databases) do
+      add :tenants, :map
```

Comment:
Always surprises me about ecto using map to represent arrays, but yeah, they recommend it hehe

Reply:
Yeah, it's because how postgres treat jsonb, I tried with the array in the result was a mess lol